### PR TITLE
TW-855 Fix errors related to Share feature

### PR DESCRIPTION
**Pull request by lendihop**
https://madfish.atlassian.net/jira/software/projects/TW/boards/4?selectedIssue=TW-855

### CODE REVIEW

File: src/modals/collectible-modal/collectible-modal.tsx [hunk 1]
```diff
@@ -222,11 +224,9 @@ export const CollectibleModal = () => {
           <Text style={styles.name}>{collectible.name}</Text>
         </View>
 
-        {isString(description) && (
-          <View style={styles.descriptionContainer}>
-            <Text style={styles.description}>{description}</Text>
-          </View>
-        )}
+        <View style={styles.descriptionContainer}>
```
Comment (svjatoslavtt) [inline src/modals/collectible-modal/collectible-modal.tsx:227]: why did you remove the condition with `isString`?

File: src/modals/collectible-modal/collectible-modal.tsx [hunk 2]
```diff
@@ -146,15 +146,17 @@ export const CollectibleModal = () => {
   }, [fa.logo]);
 
   const handleShare = useCallback(async () => {
+    const urlEncodedData = encodeURIComponent(JSON.stringify({ ...collectible, description: '', holders: undefined }));
```
Comment (svjatoslavtt) [inline src/modals/collectible-modal/collectible-modal.tsx:149]: why `description` and `holders` hardcode?

**Review by svjatoslavtt — APPROVED** [review]

File: src/modals/collectible-modal/collectible-modal.tsx (hunk 2 above)
Comment (lendihop) [reply src/modals/collectible-modal/collectible-modal.tsx:150]: we need to reduce the amount of data we send with a link, because of limits. Description is loading from the `useCollectibleInfo` and holders key isnt used on this screen, but they often contain a lot of data, thats why i choose them.
